src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -82,12 +82,11 @@
     <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.91-alpha" />
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="$(IdentityDependencyVersion)" />
     <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="$(IdentityDependencyVersion)" />
+    <PackageReference Include="Microsoft.Security.Utilities" Version="1.3.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435" PrivateAssets="all" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="$(IdentityDependencyVersion)" />
-    <PackageReference Include="Microsoft.Security.Utilities" Version="1.3.0" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
+    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="5.0.0" />
```

Comment:
I pinned this version because MSAL 2.x -> 4.x _dropped_ this dependency, so our resolved version changed from `5.0.0` to `4.7.0`. Pinning it just to reduce churn.

Reply:
👍🏼

Reply:
Can you add a comment above so others understand the dependency? I've been doing that with similar dependencies.